pypopminer/src/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+include_directories(${PROJECT_SOURCE_DIR}/src)
```

Comment:
Why do you add this?

Reply:
`set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})` was existing in your original commits, somewhere through the debugging process, I added `include_directories(${PROJECT_SOURCE_DIR}/src)` but it seems both can be removed now.  Will commit fix now.